### ISSUE TITLE
Can I use this app with server and client?

### ISSUE THREAD

**Issue by rarmando** [manthanhd/youtube-mychatappp#1]
Hello, Can I use this app with server and client using an only port? for example: Server (Windows 2012 r2) and client (Ubuntu 14.04) thanks so much...

**Comment by manthanhd** [comment]
@Rarmando Not quite. As I explained in the part 1 of the video series, this is a peer to peer app which means that it works from client to client, removing the need to have a server. You'll have to do some big changes in order to make it work with a server since its not designed to work with one.
